db/schema.rb

```diff
@@ -140,8 +140,11 @@
     t.string   "base_path",                limit: 255
     t.string   "swagger_version",          limit: 255
     t.boolean  "skip_swagger_validations",                    default: false
+    t.integer  "service_id",               limit: 8
   end
 
+  add_index "api_docs_services", ["service_id"], name: "fk_rails_e4d18239f1", using: :btree
```

Comment:
https://github.com/ankane/strong_migrations#dangerous-operations
Adding an index may cause downtime
@Martouta please create a task in 3scale/infrastructure

Reply:
Right, thanks 👍 Ready in https://github.com/3scale/infrastructure/issues/546